projects/admin/src/app/record/detail-view/budget-detail-view/acq-accounts/acq-accounts.component.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, OnInit, Input } from '@angular/core';
+import { UserService } from 'projects/admin/src/app/service/user.service';
+import { RecordService } from '@rero/ng-core';
+import { BudgetTotalService } from 'projects/admin/src/app/service/budget-total.service';
+
+@Component({
+  selector: 'admin-acq-accounts',
+  templateUrl: './acq-accounts.component.html'
+})
+export class AcqAccountsComponent implements OnInit {
+
+  /**
+   * Budget pid
+   */
+  @Input() budgetPid: string;
+
+  /**
+   * Currency code
+   */
+  @Input() currencyCode: string;
+
+  /**
+   * accounts
+   */
+  accounts: Array<any>;
+
+  /**
+   * Constructor
+   * @param userService - UserService
+   * @param recordService - RecordService
+   * @param budgetTotalService - BudgetTotalService
+   */
+  constructor(
+    private userService: UserService,
+    private recordService: RecordService,
+    private budgetTotalService: BudgetTotalService
+  ) { }
+
+  /**
+   * Init
+   */
+  ngOnInit() {
+    this.loadAcqAccounts();
+  }
+
+  /**
+   * Delete acquisition account
+   * @param acqAccountPid - string
+   * @return void
```

Comment:
useless, but in any case it should be `@returns`: https://compodoc.app/guides/jsdoc-tags.html. Needs to be checked in the whole project.

Reply:
done